.github/workflows/commit_artifacts.yml

```diff
@@ -0,0 +1,120 @@
+name: Commit Artifacts for Facebook WWW
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  download_artifacts:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.x
+      - run: npm init -y
+      - run: npm install node-fetch@2
+      - name: Download and unzip artifacts
+        uses: actions/github-script@v6
+        with:
+          script: |
```

Comment:
What do you think about moving this to `./scripts`?

Reply:
That was my original plan, but I was avoiding doing a git checkout in this first workflow so I didn't need to deal with yarn downloading a [dependency](https://github.com/facebook/react/pull/25721/files/c920b80c9849509e500ee62b65f8ad1f609b7231#diff-220ad0354283aec0db40a673841c4914bf125e3b3b113eab38ad40fe7d1e8d9dR14-R15). I figured since this is temporary I'd make it easier to delete by having it all in a single file